test/core/base/audit.js

```diff
@@ -236,6 +236,24 @@ describe('Audit', function() {
 				'https://dequeuniversity.com/rules/axe/1.0/target?application=axeAPI'
 			);
 		});
+		it('sets the lang query if locale has been set', function() {
```

Comment:
Can you add a test to make sure this is passed in from `axe.configure`? That's the API that we actually want to have direct this.